src/components/layout/debug/media-queries/index.js

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
For now, let's put everything directly in the `src/components` directory. Until we have enough complexity to warrant further structure, I say we keep it simple.